mvp.feature

```diff
@@ -1,176 +1,274 @@
-    When they verify that the information about the Journal Article Publication
+  @234
+  Scenario: User verifies information for a Publication in Journal Publication data for a publication based on a Link
+    Given that the user registers initial metadata for a Publication based on a Link
+    And the Publication is a Publication in Journal
+    When they verify that the information about the Publication in Journal Publication
     Then they see information for Volume
     And they see information for Issue
-    And they see Page Number information or an Article Number
-    # it is implicit that these fields are editable
+    And they see Page Number information
+    And they see an Article Number
+  # it is implicit that these fields are editable
 
-  Scenario: User sets the value for Publication Type for a Journal Article Publication data for a publication based on a Link
-    Given that the user associates a Publication with a Project they have previously registered a Publication
+  @274
+  Scenario: User sets the value for Publication Type for a Publication in Journal Publication
+    Given that the user registers initial metadata for a Publication based on a Link
     And have clicked Next
     And they are on the screen for registering information about the Publication Reference
     When they select a Reference Type from the list
-        | Journal Article |
+      | Publication in Journal |
     And they select a Publication Subtype from the list
-        | Article              |
-        | Short communication  |
-        | Leader               |
-        | Letter to the editor |
-        | Review               |
+      | Article              |
+      | Short communication  |
+      | Leader               |
+      | Letter to the editor |
+      | Review               |
     And they verify a (non-obligatory) DOI from a Publication based on a Link
     And they verify information for Journal for a publication based on a Link
-    And they verify (non-obligatory) information for a Journal Article Publication data for a publication based on a Link
+    And they verify (non-obligatory) information for a Publication in Journal Publication data for a publication based on a Link
     And they select a value for Peer-review
-    And they upload a (non-obligatory) Author Agreement
-    Then they see that the Next button is enabled
+    And they view the Summary Page
+    Then they see that the information is registered
+
+  @392
+  Scenario: User sets the value for Publication Type for a Book
+    Given that the user registers initial metadata for a Publication based on a Link
+    And have clicked Next
+    And they are on the screen for registering information about the Publication Reference
+    When they select a Reference Type from the list
+      | Book |
+    And they select a Publication Subtype from the list
+      | Monography |
+      | Anthology  |
+    And they verify information for Publisher for a publication based on a Link
```

Comment:
refereres det til et scenario her? hvor er det scenariet isåfall?
hvis ikke; kan det skrives som `And they verify that the information for the Publisher is as expected/correct/the same that is listed in the original Link`?